/wust_arm_driver:
  ros__parameters:
    debug: true
    use_fake_hardware: true
    goal_tolerance: 0.01
    goal_timeout: 5.0
    controller_freq: 50
    device_name: /dev/ttyUSB0
    baud_rate: 115200
    flow_control: none
    parity: none
    stop_bits: "1"
    
    # 自定义驱动参数
    update_rate: 50 # 发布 joint_states 的频率 (Hz)
    joint_names: ["joint1", "joint2", "joint3", "joint4", "joint5", "joint6", "joint7"]